proposals/scopes.md

```diff
@@ -366,38 +462,26 @@ A|    var _x = 1;
 
 ```
 Start Generated Range C0 { // A
-  field flags: has definition, is scope
-  definition: file.js, scope offset 0
-  callsite: none
-  bindings: x -> _x, z -> _z
+  field flags: has definition
+  definition: scope start 0
 }
-;
+Bindings [x -> _x, z -> _z]
 Start Generated Range C16 { // B
-  field flags: has definition, is scope
-  definition: file.js, scope offset 1
-  callsite: none
-  bindings: message -> _m, y -> _y
+  field flags: has definition, is stack frame
+  definition: scope start 1
 }
-;
-;
-;
+Bindings [message -> _m, y -> _y]
 End Generated Range C1 // B
-;
 Start Generated Range C0 { // C
   field flags: has definition, has callsite
```

Comment:
In this example, I think we can delete the "has callsite" bit which isn't used anymore right?

Reply:
Indeed, removed!